.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Initial reformat with Prettier
+4b8c8835f8916cc7c63e74225c28bfe0502a9dac
```

Comment:
Awesome! I didn't know this file "was a thing."

References:
- https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view
- https://moxio.com/blog/ignoring-bulk-change-commits-with-git-blame/